README.md

```diff
@@ -22,13 +22,31 @@ We want to make a self-hosted workflow manager for companies and biotech labs th
 
 Flow by LabGrid is currently under active development. Development docker containers that support hot code reloading on edit are provided.
 
-## First, setup your environment.
+
+## First, prepare authentication settings
+
+At the moment, only Auth0 is supported; you will need an Auth0 account.
+
+1. Setup a react application for your auth0 tenant.  
+1. And an API.  
+1. Add the following permission scopes to the API: 
+`write:protocols`, `read:protocols`, `write:runs`, `read:runs`  
+1. Add a redirect url and allowed origin to the auth0 application: `http://localhost:3000`.
```

Comment:
Should these start with `1.`, `2.`, `3.`, ...?

Reply:
You mean why did I use `1.` for each?